modules/test/playwright/tests/frontend-data-set-views-web/actions.spec.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {dataSetManagerPagesTest} from './fixtures/dataSetManagerPageTest';
+import {loginTest} from '../../fixtures/loginTest';
+
+export const test = mergeTests(dataSetManagerPagesTest, loginTest);
+
+test('There are no item actions created in the Action tab', async ({
+	dataSetManagerActionsPage,
+	dataSetManagerViewsPage,
+	page,
+}) => {
+	await dataSetManagerViewsPage.goto();
```

Comment:
Does this make sense? :thinking: 

Why do you navigate to two different pages without doing anything in the middle?

Reply:
Because `dataSetManagerViewsPage` is the main window and `dataSetManagerActionsPage` is a tab of it, maybe?

Reply:
In that case I would expect `dataSetManagerViewsPage` to have a `gotoActionsTab`, but well, your pattern may also make sense, so not sure...

Additionally, if `dataSetManagerViewsPage` had a `gotoActionsTab` (my expectation, not the code in the PR), where would the `dataSetManagerActionsPage` come from? Would `gotoActionsTab` return it perhaps? :thinking: 

What do others think? What do you like more / find more intutive?  @dsanz @bryceosterhaus @markocikos

Reply:
`dataSetManagerActionsPage.goto()` to me represents full navigation, from home page to the actions page. Internally in its implementation, it can contain `dataSetManagerViewsPage.goto()`, followed by click on relevant locator. In this specific case, it should probably internally contain `dataSetManagerViewDetailsPage.goto()`, since you always first open the details page.

Reply:
That point comes from code that is going to be deprecated for the moment (creating data in page fixtures), so I'll close it